conf/cassandra.yaml

```diff
@@ -647,6 +647,17 @@ seed_provider:
       # based on the configured name service on the system. Defaults to "false".
       #  resolve_multiple_ip_addresses_per_dns_record: "false"
 
+
+#  - class_name: org.apache.cassandra.locator.UrlSeedProvider
+#    parameter:
+        # seeds_url points to config which container one seed per line.
+        # Ex: "http://<seeds-url-address>"
+#      - seeds_url: "http://127.0.0.1"
```

Comment:
I think it should be a comma separated list of URLs. Having one URL is a single point of failure. Users should be able to set as many points as they like.

Reply:
Indeed, `metadata_url` or `url` is better. My point is that a URL adds a single point of failure to the cluster, which is not good (without reading the seed it won't start, right?). We can easily support a list of URLs, which should be fine and safe.